docs/hero-banner.md

```diff
@@ -12,11 +12,18 @@ Here are some cool activities to get you started with your @boardname@!
 * url: https://arcade.makecode.com/#tutorial:/tutorials/intro
 * cardType: link
 ---
-* name: Save the Turkeys!
+* name: Send a Snow Globe
 * buttonLabel: Code Now
-* description: Create an amazing game to share!
+* description: New! Share an animated snow globe greeting card.
+* imageUrl: /static/hero-gallery/snowglobes.png
+* url: https://arcade.makecode.com/#tutorial:/tutorials/snow
+* cardType: link
```

Comment:
```suggestion
* url: /tutorials/snow
* cardType: tutorial
```

as a link, it'll open in a new tab, but as tutorial we keep to single tab which is nice